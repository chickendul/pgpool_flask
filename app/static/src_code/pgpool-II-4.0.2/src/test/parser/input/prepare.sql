version https://git-lfs.github.com/spec/v1
oid sha256:83288986c4a6b9e67156b687dffbd1938fda02409b923f0aef4f49b494edb346
size 122
