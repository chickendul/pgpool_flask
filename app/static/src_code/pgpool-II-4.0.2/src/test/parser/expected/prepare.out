version https://git-lfs.github.com/spec/v1
oid sha256:d2fae8cf0c5dc1a6bfd3a21571c1439710e25c212a4efcf3b19a8f1cf7b150ec
size 118
